workflow/realisations.py

```diff
@@ -363,6 +364,22 @@ class SRFConfig(RealisationConfiguration):
     resolution: float
     """The resolution of the SRF geometry"""
 
+    # The following attributes are only used for the point source approximation
+    stype: Optional[schemas.Stype] = None
+    """Slip type for generic_slip2srf"""
+
+    risetime: Optional[float] = None
+    """Rise time for generic_slip2srf"""
+
+    risetimefac: Optional[float] = None
+    """Rise time factor for generic_slip2srf"""
+
+    risetimedep: Optional[float] = None
+    """Rise time depth dependency for generic_slip2srf"""
+
+    inittime: Optional[float] = None
+    """Initial time for generic_slip2srf"""
```

Comment:
Apparently `Optional` is deprecated now? Or, at least, ruff keeps nagging me about it. Better to use 


```suggestion
    stype: schemas.Stype | None = None
    """Slip type for generic_slip2srf"""

    risetime: float | None = None
    """Rise time for generic_slip2srf"""

    risetimefac: float | None = None
    """Rise time factor for generic_slip2srf"""

    risetimedep: float | None = None
    """Rise time depth dependency for generic_slip2srf"""

    inittime: float | None = None
    """Initial time for generic_slip2srf"""
```
for new code at any rate.